### ISSUE TITLE
'react-powerbi' is not in the npm registry.

### ISSUE THREAD

**Issue by jpmazza** [Microsoft/PowerBI-React#5]
Registry returned 404 for GET on https://registry.npmjs.org/react-powerbi
npm ERR! 404 
npm ERR! 404 'react-powerbi' is not in the npm registry.

**Comment by habelmarottipuzha** [comment]
@jpmazza Did you find a solution to work with.I'm also getting same error. Didn't found any references yet

**Comment by MacsDickinson** [comment]
This still isn't available on npm. @mattmazzola has this been published?

**Comment by mattmazzola** [comment]
No, it has not been published because the repo is not complete.  I only had 2 days to get this and the associated repo: https://github.com/Azure-Samples/powerbi-react-client setup and I did not finish before I changed roles/jobs.  I do not work on Power BI anymore. I opened issues that need to be fixed before this can be published so the new maintainers could continue the work but I see they have not.

See: #3

**Comment by noanu** [comment]
Hi @MacsDickinson ,
We (the new maintainers of PowerBI embedded) are aware of the problem you raised. We are making effort to fix the issues that mattmazzola mentioned, and will update you on the status soon.

**Comment by emzero** [comment]
Is this project being maintained or has been abandoned?

**Comment by whitneyland** [comment]
This is such bullshit.  Why on earth would someone not update the read-me say the project is in an unfinished state?

We know people are busy.  Just add "WIP do not use yet" and it will save everyone time.

**Comment by ucaboro** [comment]
@noanu has there been any progress on this? If not do you have a reccomended way of connecting power bi to React?

**Comment by howlowck** [comment]
I published a working version of the package on npm

**Comment by nicowernli** [comment]
@howlowck so this issue could be closed, right?

**Comment by iwoork** [comment]
@howlowck any documentation on how to use?

**Comment by howlowck** [comment]
@iwoork I added "how to use" code to the readme on https://github.com/howlowck/react-powerbi.  If you have more questions, please ask on that repo.
